### PR TITLE
[FIX] base: tours are getting closer to browser_js timeout limit

### PR DESCRIPTION
**Pull request by sts-odoo**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/tests/common.py [hunk 1]
```diff
@@ -1498,7 +1498,7 @@ def _wait_code_ok(self, code, timeout, error_checker=None):
 
     def navigate_to(self, url, wait_stop=False):
         self._logger.info('Navigating to: "%s"', url)
-        nav_result = self._websocket_request('Page.navigate', params={'url': url}, timeout=20.0)
+        nav_result = self._websocket_request('Page.navigate', params={'url': url}, timeout=40.0)
```
Comment (Xavier-Do) [inline odoo/tests/common.py:1501]: 20 seconds looks already to much to load a page especially now that assets are pregenerate. Do you have an example of a test close to this limit?

File: odoo/tests/common.py [hunk 2]
```diff
@@ -1764,7 +1764,7 @@ def authenticate(self, user, password):
 
         return session
 
-    def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=None, error_checker=None, watch=False, **kw):
+    def browser_js(self, url_path, code, ready='', login=None, timeout=120, cookies=None, error_checker=None, watch=False, **kw):
```
Comment (Xavier-Do) [inline odoo/tests/common.py:1767]: Do you have example of tests longer than that? Shouldn't the parameter be given by those tests?

File: addons/website/tests/test_lang_url.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def test_04_url_cook_lang_not_available(self):
         """
         # 1. Load backend
         self.authenticate('admin', 'admin')
-        r = self.url_open('/web')
+        r = self.url_open('/web', timeout=20)
```
Comment (rdeodoo) [inline addons/website/tests/test_lang_url.py:58]: ?

Comment (sts-odoo) [reply addons/website/tests/test_lang_url.py:58]: it takes 14 seconds on odoo.sh, so default value of 12 is a bit low

Comment (rdeodoo) [reply addons/website/tests/test_lang_url.py:58]: If the web client is taking too long to open in post install tests in odoo.sh, shouldn't you fix it also on other tests doing an url_open of `/web`?
And actually, shouldn't this be part of `url_open` to increase the timeout automatically if the given url / target is /web, so caller don't have to think about it?
That would probably be better:
- right now, there is at least 2 other tour doing it also (that I can easily grep), aren't those 2 also failing?
- the next tour we introduce which does a url_open of /web, we won't think about this 20 second for odoo.sh case, you will end up with the same issue.

AFAIC, I'd really "improve" the url_open definition to increase it when loading the backend, maybe even stating that the main reason is for odoo.sh, but maybe wait for @Xavier-Do opinion.

Comment (Xavier-Do) [reply addons/website/tests/test_lang_url.py:58]: I will investigate why this url_open is slower. /web should actually take around 1 second when cold if the assets are pregenerate

Comment (Xavier-Do) [reply addons/website/tests/test_lang_url.py:58]: On runbot, split
```
2022-09-30 10:16:47,369 13 INFO 19801767-16-0-all odoo.addons.website.tests.test_lang_url: Starting TestControllerRedirect.test_04_url_cook_lang_not_available ... 
2022-09-30 10:16:47,686 13 INFO 19801767-16-0-all odoo.addons.base.models.res_users: Login successful for db:19801767-16-0-all login:admin from n/a 
2022-09-30 10:16:47,697 13 INFO 19801767-16-0-all odoo.addons.base.models.ir_http: Generating routing map for key 1 
2022-09-30 10:16:49,785 13 INFO 19801767-16-0-all werkzeug: 127.0.0.1 - - [30/Sep/2022 10:16:49] "GET /web HTTP/1.1" 200 - 128 0.068 2.026
2022-09-30 10:16:50,527 13 INFO 19801767-16-0-all werkzeug: 127.0.0.1 - - [30/Sep/2022 10:16:50] "GET /contactus HTTP/1.1" 200 - 214 0.062 0.672
```

Locally
image


On runbot, during nighlty:
2022-09-30 06:54:33,506 13 INFO 19776230-16-0-all_no_autotag odoo.addons.website.tests.test_lang_url: Starting TestControllerRedirect.test_04_url_cook_lang_not_available ... 
2022-09-30 06:54:33,808 13 INFO 19776230-16-0-all_no_autotag odoo.addons.base.models.res_users: Login successful for db:19776230-16-0-all_no_autotag login:admin from n/a 
2022-09-30 06:54:33,819 13 INFO 19776230-16-0-all_no_autotag odoo.addons.base.models.ir_http: Generating routing map for key 1 
2022-09-30 06:54:36,523 13 INFO 19776230-16-0-all_no_autotag werkzeug: 127.0.0.1 - - [30/Sep/2022 06:54:36] "GET /web HTTP/1.1" 200 - 128 0.073 2.635
2022-09-30 06:54:37,219 13 INFO 19776230-16-0-all_no_autotag werkzeug: 127.0.0.1 - - [30/Sep/2022 06:54:37] "GET /contactus HTTP/1.1" 200 - 214 0.068 0.621

TODO check design theme

Comment (Xavier-Do) [reply addons/website/tests/test_lang_url.py:58]: Fix proposal here #101780

**Comment by rdeodoo** [comment]
The fix of @Xavier-Do should have fixed the issue another way, can we close?

**Pull request closed without merging**